decompiler/ObjectFile/ObjectFileDB.h

```diff
@@ -185,6 +185,8 @@ class ObjectFileDB {
   void analyze_functions_ir2(
       const fs::path& output_dir,
       const Config& config,
+      const std::optional<std::function<void(std::string)>> prefile_callback,
+      const std::optional<std::function<void()>> postfile_callback,
```

Comment:
`std::function` already has the ability to be set or unset, so the `optional` isn't really needed. By default it is unset, and you can just check with something like:
```cpp
if (postfile_callback) {
  postfile_callback();
}
  ```